src/main/webapp/3Dscript.js

```diff
@@ -16,10 +16,54 @@ const poseTransform = {
   /* Unit: 4 meters*/ translateZ: 0,
   /* Unit: degrees*/ rotate: 0,
   /* Scalar*/ scale: 1};
+
+const poseStart = {start: 0};
+const poseEnd = {end: 10000};
```

Comment:
Consider using maximum int number or Number.POSITIVE_INFINITY instead of 10000.

Reply:
thank you, i wanted to use pose.length but it didnt work